aspnetcore/test/troubleshoot-azure-iis.md

```diff
@@ -137,6 +137,22 @@ The error message lists all the installed .NET Core versions and the version req
 
 When running in development (the `ASPNETCORE_ENVIRONMENT` environment variable is set to `Development`), the specific error is written to the HTTP response. The cause of a process startup failure is also found in the Application Event Log.
 
+This error can also occur if the asp.net core app is using in-process hosting model, and aspnetcore module logging is enabled through the stdoutLogFile, to log the stdout files to WWWROOT folder (or the application content folder). 
```

Comment:
I think this can be condensed a bit to say "This error can also occur if there is an error with configuration, for example, if there is an invalid path to a log folder." The information below is redundant IMO.

Also, can you give more details about the wwwroot issue? I don't necessarily believe wwwroot is an issue; I thought it was only with an invalid path.

Reply:
Why would it choke on writing to any folder in the IIS physical app path?

@fasigpt Can you provide the exact steps to repro this (with a valid path)?

Reply:
Confirmed ... I can't repro this behavior with IIS local or Azure Apps. For example ...

<img width="256" alt="Capture" src="https://user-images.githubusercontent.com/1622880/69378486-1964d680-0c74-11ea-8595-ce29da832f08.PNG">